### ISSUE TITLE
Cooperation: New instruction slides

### ISSUE THREAD

**Issue by LavalleyCA** [laureate-institute-for-brain-research/StimToolOnline#63]
[Cooperation_Task_Instructions.11.29.2022.pptx](https://github.com/laureate-institute-for-brain-research/StimToolOnline/files/10124710/Cooperation_Task_Instructions.11.29.2022.pptx)

No new audio needed.

**Comment by LavalleyCA** [comment]
Hi! Lots of updates:

- updated slides (audio to come): We changed every "meaningless" (text underlined) to be "neutral" (text in blue). The only change to your code should be the message that is displayed at the bottom of the chatrooms (i.e. instead of "You will be shown the meaningless (text underlined) image unless the person you choose decides to help you", it should say "You will be shown the neutral (text in blue) image unless the person you choose decides to help you")
- outcome media schedule: The order of images is now fixed (included the neutral images which are no longer scrambled). There are only 3 tones now - positive, negative, and neutral
      - I had to invent paths for things that didn't exist/I didn't know where they went so hopefully that stuff is clear

[outcome_media_schedule.csv](https://github.com/laureate-institute-for-brain-research/StimToolOnline/files/10158385/outcome_media_schedule.csv)

[Cooperation_Task_Instructions.12.5.2022.pptx](https://github.com/laureate-institute-for-brain-research/StimToolOnline/files/10158367/Cooperation_Task_Instructions.12.5.2022.pptx)
